lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -191,6 +166,47 @@ def self.is_firebasecmd_supported?(cmd)
 
         true
       end
+
+      def self.get_token
+        client = Signet::OAuth2::Client.new(
+          token_credential_uri: 'https://oauth2.googleapis.com/token',
+          client_id: FirebaseAppDistributionLoginAction::CLIENT_ID,
+          client_secret: FirebaseAppDistributionLoginAction::CLIENT_SECRET,
+          refresh_token: ENV["FIREBASE_TOKEN"]
+        )
+
+        client.fetch_access_token!
+        return client.access_token
+      rescue Signet::AuthorizationError => error
+        UI.crash!("Wrong value for FIREBASE_TOKEN")
+      end
+
+      def self.get_app(params)
+        token = get_token
+
+        # begin
+        appId = (params[:app]).to_s
```

Comment:
Rename to `app_id`.

Also, no need to call `to_s` here. `app_id` is only used in interpolated strings, and ruby automatically calls `to_s` on values used in string interpolation.